app/views/games/board.html.erb

```diff
@@ -10,45 +11,24 @@
   -->
 
   <table>
-    <% 4.times do %>
+    <% 8.times do %>
       <tr>
-        <% 4.times do %>
-          <td><div class="white"><%= col[col_index] + (row_index + 1).to_s %><p><%= @starting_positions.key([row_index, col_index]) %></p></div></td>
-          <% col_index += 1 %>
-          
-          <td><div class="black"><%= col[col_index] + (row_index + 1).to_s %><p><%= @starting_positions.key([row_index, col_index]) %></p></div></td>
+        <% 8.times do %>
+          <td>
+            <div><%= col[col_index] + (row_index+1).to_s %><p><%= %></p></div>
+            <% piece = get_piece_at(col_index, row_index) %>
+            <% if piece %> 
+              <div><%= piece.color %></div>
+              <div><%= piece.type %></div>
```

Comment:
After removing ChessPiece.type, you can use Object.class to determine what type a piece is

`<div><%= piece.class %></div>`

[Determining type of an object in ruby](http://stackoverflow.com/a/15769829/3935097)